test/battle/move_effect/salt_cure.c

```diff
@@ -0,0 +1,71 @@
+#include "global.h"
+#include "test/battle.h"
+
+ASSUMPTIONS
+{
+    ASSUME(gBattleMoves[MOVE_SALT_CURE].effect == EFFECT_SALT_CURE);
+}
+
+SINGLE_BATTLE_TEST("Salt Cure inflicts 1/8 of the target's maximum HP as damage per turn")
+{
+    GIVEN {
+        PLAYER(SPECIES_WOBBUFFET);
+        OPPONENT(SPECIES_WOBBUFFET);
+    } WHEN {
+        TURN { MOVE(player, MOVE_SALT_CURE); }
+        for (i = 0; i < 3; i++)
+            TURN {}
+    } SCENE {
+        s32 maxHP = GetMonData(&OPPONENT_PARTY[0], MON_DATA_MAX_HP);
+        ANIMATION(ANIM_TYPE_MOVE, MOVE_SALT_CURE, player);
+        MESSAGE("Foe Wobbuffet is being salt cured!");
+        for (i = 0; i < 4; i++)
+            ANIMATION(ANIM_TYPE_GENERAL, B_ANIM_SALT_CURE_DAMAGE, opponent);
+            HP_BAR(opponent, damage: maxHP / 8);
+            MESSAGE("Foe Wobbuffet is hurt by Salt Cure!");
```

Comment:
Based on the identation, I think you wanted to add braces here.

Reply:
Fixed.